### ISSUE TITLE
Raumbox

### ISSUE THREAD

**Issue by LukasFreeze** [comakingspace/do-something#309]
Wir bekommen Material für eine "Raumbox" mit Fenster!

![IMG_20240209_134630_629](https://github.com/comakingspace/do-something/assets/19755883/1150f653-3e47-4d72-8191-d334f69fd13a)

*notfalls* ließe sich das ganze Holz auch anderweitig sinnvoll einsetzen, aber in der Telegram-Gruppe gab es schon mehrere Vorschläge für einen Wiederaufbau als "Raum im Raum".
- Wärmekabine im Winter
- Dunkelkammer
- Sprühkammer
- Siebdruck
- ...weitere Ideen?

Neben der Verwendung (aber evtl. davon abhängig?) müssten wir einen geeigneten Ort dafür finden.
- Maße: Urzustand ca. 2x2 m, könnte mit unseren Möglichkeiten natürlich leicht angepast werden :-)

Und last but not least: wer kümmert sich um den Aufbau? Wir brauchen ein Team, bitte meldet euch!

**Comment by LukasFreeze** [comment]
Ich bin der Meinung, dass wir zumindest eine Zweitnutzung als Wärmekabine anpeilen sollten.

Im Gemeinschaftsraum fiele mir dazu der tote Bereich links vom Eingang ein, die "Theke" könnte ja auch auf die andere Seite. Das ist aber nur mal laut gedacht und noch nicht vor Ort geprüft...

**Comment by NitramLegov** [comment]
@Johannes09 und ich haben heute über genau das Thema auch schon gesprochen.

Unser Vorschlag ist folgendes:
Wir nutzen einen Raum im Raum primär, um im Winter Arbeiten zu ermöglichen, die eine warme Umgebung erfordern. Auf den ersten Blick sind das:
- Nähen
- Löten & andere Basteleien mit Elektronik

Beides sind Aktivitäten, bei denen man seine Hände braucht, Handschuhe aber keine Option sind.

Daher sollten wir den Raum im Raum groß genug machen, um die Nähmaschine und *einen* Elektronikarbeitsplatz aufzunehmen. Zusätzlich sollte natürlich eine Wärmequelle rein (zum Beispiel ein Heizpilz).

Als Ort bietet sich im E-Room die Wand zum Nachbarn an. Wenn wir den Raum von der Wand bis zum Ende des ersten Schreibtischs aufbauen sollten wir genug Platz haben und gleichzeitig klein genug sein, damit ein Heizpilz ausreicht.

**Comment by LukasFreeze** [comment]
Die Stelle im eRoom könnte ich mir auch gut vorstellen :-)

Denke aber, dass es bessere Heizoptionen gibt, weil ein Heizpilz selbst
recht viel Platz braucht und die Wärme für einen Sitzarbeitsplatz gerne von
unten kommen darf.

Axel hat das Material gestern vorbeigebracht und wird hier noch Fotos vom
Abbau posten, damit klar wird wie alles zusammengehörte - aktuell steht das
Fenster schon einmal im eRoom und Holz/Alu (beschriftet mit "#309") auf dem
gelben Gitterwagen neben dem Restholzlager

**Comment by NitramLegov** [comment]
Johannes und ich haben uns gestern mal die Situation vor Ort angeschaut.

Wir schlagen eine Höhe von 2,5m vor. Das ist etwas höher als das Regal, das gerade an der Wand steht :)

Als tiefe in den Raum hinein schlagen wir 1,7m vor. Damit geht die Box bis zum Ende des Schreibtisches, auf dem aktuell das Pi-Lab steht. Das könnte dann natürlich umziehen (oder abgebaut werden...wurde das jemand genutzt?)

Bezüglich der Breite sind wir durch den Verteilerkasten limitiert. Vor dem Verteilerkasten muss nach Johannes Recherchen mindestens 70 cm Platz bleiben. Um etwas mehr Platz zu lassen schlagen wir vor, den Raum kurz nach dem Kabelschacht enden zu lassen (der Kabelschacht wäre also IM Raum). Ich glaube das waren dann 2,7m...muss man nochmal messen 🙈

Wir müssten also auf jeden Fall noch Holz dazukaufen. Und wir sollten entscheiden, ob wir eine solche Box dämmen möchten.


Bezüglich der Heizung: Ich hatte einen Elektro Heizpilz im Kopf, die sollten nicht soooo viel Platz verbrauchen:
https://www.amazon.de/Enders-Terrassenheizer-4906-h%C3%B6henverstellbar-Infrarotstrahler/dp/B00ANTQSCM
https://www.amazon.de/Enders-Deckenstrahler-Marbella-Elektro-Deckenstahler/dp/B082XR6ZJP

Ich könnte mir auch gut vorstellen, an den beiden Arbeitsplätzen jeweils über und unter dem Tisch ein kleines Infrarotpanel anzubringen und zusätzlich einen Heizpilz aufzustellen. Getrennt schaltbar und jeweils mit einer automatischen Abschaltung nach Zeit x versehen könnte der Heizbedarf gezielt zu und abgeschaltet werden :-)

**Comment by nawhcs** [comment]
Hier noch ein paar Bilder vom Abbau, damit sollte die Struktur besser Erkennbar sein. Das Gefache kann natürlich auch besser / anders ausgelegt werden beim Aufbau.

![IMG_7043](https://github.com/comakingspace/do-something/assets/134232828/24a4756d-5bea-4b14-8187-65d6d5beeaf9)
![IMG_7044](https://github.com/comakingspace/do-something/assets/134232828/fbeae33f-7ed9-46c7-9cff-123b298813ee)
![IMG_7045](https://github.com/comakingspace/do-something/assets/134232828/f43b2066-36c3-49aa-8d89-0f0d94445cd4)
![IMG_7046](https://github.com/comakingspace/do-something/assets/134232828/6d322baa-12cf-4938-ada6-cbb06a2ac3ee)
![IMG_7047](https://github.com/comakingspace/do-something/assets/134232828/f0f6e5e6-6c3a-4d05-9ec7-64437611a9f3)
![IMG_7048](https://github.com/comakingspace/do-something/assets/134232828/ad44bb18-84c6-4203-ae4b-47eab599e0f6)
![IMG_7049](https://github.com/comakingspace/do-something/assets/134232828/4cc0fe1c-8fed-47b5-b25a-6bb9fbfdf2ef)
![IMG_7051](https://github.com/comakingspace/do-something/assets/134232828/87c27e66-cbfb-4c9c-ad46-c701962e19d9)

**Comment by LukasFreeze** [comment]
@FHuchinson und ich haben hier mal angefangen und das Regal von der Rückwand umgebaut :-)

![20240721_025433](https://github.com/user-attachments/assets/e21572d1-46cc-4aa3-8fef-d7051d5965b0)

Leider waren wir nur zu zweit und vor allem nicht so die Elektronik-Experten, daher haben wir zwar aufgeräumt so gut es ging aber auch ein paar "Krimskrams-Kisten" so wie sie waren an einen anderen Platz gestellt.
- Dronen/RC und Deltadruckerteile auf eine Palette neben den Fahrzeugplätzen
- Material für "Elektro-Installation" unter den Schaltschrank
- beschriftete Kisten ins Regal zurück
- unklar sortierte Kisten tendenziell eher in die neu entstandene Nische neben dem Laser-PC

Ich würde vorschlagen, dass in das große Regal nur noch sortierte und beschriftete Kisten kommen und Neuzugänge/Übergangslösungen weiter hinten versteckt werden bis sie sich jemand vornehmen konnte. Da müssen aber Leute helfen, die sich besser auskennen ;-)

**Comment by LukasFreeze** [comment]
Ich habe dann noch das Holz rübergefahren und mit dem Layout begonnen, kann aber noch nicht einschätzen was wir noch kaufen müssen. Irgendwann war dann auch mal gut ;-)
![20240721_022131](https://github.com/user-attachments/assets/ede47029-342f-4018-8c40-c5baa13c88f1)

**Comment by LukasFreeze** [comment]
Fortschritt mit @Johannes09 🥳
![20240801_030048](https://github.com/us habeer-attachments/assets/016d274b-7ebb-4b81-bf58-853025ce816b)

Wir haben uns für eine materialsparende Bauweise ohne doppelten Boden entschieden. Die "Tür" könnte zumindest für die erste Saison einfach ein schwerer Vorhang werden, dann sehen wir ob das reicht 🤔

**Comment by zephron25** [comment]
Hat gestern viel Spaß gemacht, hier weiterzuarbeiten. Irgendwann hatten wir einfach kein Material mehr 🤓

Das Fenster haben wir anders eingebaut als im Original. Genauer gesagt haben wir kein Rähmchen außen angebracht. Stattdessen haben wir die Latten abgehobelt und so angebracht, dass sie in die Aussparungen am Fenster passen (oben und unten):

![Image](https://github.com/user-attachments/assets/96850a98-3752-4c32-8dec-b880d1767341)
![Image](https://github.com/user-attachments/assets/e089cfd7-3553-49c8-86b8-f2f46a34fe7a)
![Image](https://github.com/user-attachments/assets/77cbd645-f20d-49d4-8cec-a9a789e2715f)

Da ich früher gehen musste, bin ich gespannt, wie der Zwischenstand jetzt genau aussieht 😊

**Comment by LukasFreeze** [comment]
Ich habe vier Dämmstoffmatten mitgebracht, die bei meinem Trockenbauprojekt übrig geblieben sind :-)

![Image](https://github.com/user-attachments/assets/90afc587-c237-4e28-81d0-5d5293a223bc)

Wie viel wir dämmen könnten wir denke ich nach und nach entscheiden, aber die Decke kommt mir schonmal sinnvoll vor.

**Comment by LukasFreeze** [comment]
Ich habe von der Idee gehört, den großen Inkjet-Drucker zum Schutz vor Frost in die Raumbox zu stellen und bin schwer dagegen: es sollte ein flexibel nutzbarer Raum werden und kein "Gehäuse" für eine einzelne Maschine, die fast den gesamten Platz beansprucht...

**Comment by LukasFreeze** [comment]
Thema Heizung: ich bin für ein Infrarotpanel, das schnell und nur bei Bedarf die daneben sitzende Person aufwärmt, aber wenig an die Luft verschwendet.

Zum Einstieg in genauere Recherche vllt dieser Artikel: https://www.chip.de/artikel/Die-besten-Infrarotheizungen-im-Vergleich_184372330.html

Zur dort empfohlenen Könighaus P-1000 sollte ich nach diesem WE auch einen eigenen Erfahrungsbericht liefern können :-)

**Comment by NitramLegov** [comment]
+1 für Infrarot. Es geht nicht darum, einen warmen Raum zu haben sondern darum, Menschen aufzuwärmen :)

Für mich ist auch die Frage offen, ob im Raum ein Arbeitsplatz vorhanden sein soll. In diesem Fall schlage ich vor, mehrere Infrarot Paneele zu holen, um z.B. Oberkörper und unterkörper gleichzeitig wärmen zu können.

**Comment by LukasFreeze** [comment]
Also ich habe zuhause gerade die Könighaus P-800 (hatte mich beim letzten Post im Modell geirrt) in Betrieb genommen und sie wird gut warm, auch mit einem Meter Abstand :-)

Ich denke dass eine reicht, etws über dem Boden hochkant angebracht wärmt sie im Sitzen locker den kompletten Körper und bei Steharbeiten ist man obenrum idR sowieso aktiv (=warm) und sogar da würde man noch etwas vom Infrarot spüren. Falls wir trotzdem direkt zwei holen wäre ich eher für zwei getrennt schaltbare Plätze.

Was unbedingt sein sollte ist (pro Einheit!) eine große Statusleuchte Timer der zB immer nach 10 min abschaltet, gibt es bestimmt als Vorschaltgerät? Einer Home-Assistant oder ähnliche Lösung fände ich kritisch, weil wir damit im Fehlerfall richtig viel Geld verheizen 😜

Edit: so etwas in der Art --> https://www.amazon.de/Brennenstuhl-Schaltsteckdose-integrierter-Timer-Steckdose-Ber%C3%BChrungsschutz/dp/B0963HB7ND

**Comment by NitramLegov** [comment]
Ich stimme dir zu, dass wir eine vorgeschaltete Steckdose mit Timer brauchen, damit das Teil eine maximale Laufzeit hat.

Auch dabei, dass der Timer nicht über HomeAssistant kontrolliert werden sollte stimme ich dir zu. Gleichzeitig finde ich eine Lösung, die zwar die Timerfunktion direkt in der Steckdose abdeckt, aber dennoch in HomeAssistant integriert ist schon gut. Damit ließe sich dann auch herausfinden, ob das Gerät schon sehr lange läuft (bspw. weil es mehrfach hintereinander angemacht wurde..)

Sowas zum Beispiel würde das abdecken: https://www.shelly.com/de/products/shelly-plug-s-gen3

**Comment by NitramLegov** [comment]
Heizung ist bestellt - sollte die Tage kommen.

**Comment by NitramLegov** [comment]
Die Heizung ist angekommen. Wir brauchen jetzt also eine Lösung, damit die Laufzeit beschränkt wird :)

Im Space haben wir noch einige von diesen zwischensteckern:
https://devices.esphome.io/devices/nous-a1t/

Die laufen aktuell noch auf tasmota. Ich denke für unsere Verwendung sollten wir sie auf esphome umflashen.

Alternativ können wir auch Hardware bestellen, dann muss ich nur wissen, was genau 😊

**Comment by Pakue95** [comment]
Würde da etwas nicht digitales zur Steuerung nehmen wie ein Treppenhaus Lichtschalter. 20min sollte ja passen für die Anwendung. 

https://www.pollin.de/p/finder-treppenlicht-zeitschalter-14-01-8-230-0000-340669

**Comment by LukasFreeze** [comment]
Ich finde eine mechanisch ablaufende Uhr auch gut!

Auf Telegram wurde gestern noch ein Münzautomat ins Spiel gebracht, Jonah könnte sogar einen spenden. Die Idee, einen (geringen!) Betrag zahlen zu müssen mag ich prinzipiell um den bewussten Umgang mit dem beträchtlichen Strombedarf zu fördern, allerdings könnten wir genauso gut sagen etwas Wärme gehört zu den Grundlagen die der Space eben übernimmt. Wichtig wäre da wohl vor allem @NitramLegov s Einschätzung zum organisatorischen Aufwand einer weiteren Kasse, und bisher haben wir noch keine Details zu dem angebotenen Gerät.

**Comment by NitramLegov** [comment]
Was ist der Vorteil einer nicht digitalen gegenüber einer digitalen Lösung?

Ich sehe den Vorteil, das in den sicherungskasten zu integrieren. Das ist aber unabhängig von digital oder analog. Ansonsten sehe ich beim Thema digital/analog wenig Vorteile für analog.

Im Gegenteil: ich vermute, wir haben mehr Leute, die im Zweifel eine digitale Lösung anpassen (können) als eine analoge. 

Eine digitale Lösung könnten wir überwachen, um ein Gefühl zu bekommen, wie lange das Teil läuft. Bei einer analogen Lösung müssten wir das wahrscheinlich dazubasteln.

Bezüglich Münzautomat: Der wäre ja ein *Zusatz* zur ablaufenden Uhr :) quasi anstelle eines Schalters ein münzautomat.

Der organisatorische Aufwand, noch eine Kasse zu zählen sollte nicht allzu hoch sei. Allerdings sehe ich Nachteile, falls mal jemand keine oder nicht die passenden Münzen dabei hat 🙈 die person könnte dann ja keine Wärme nutzen. Fände ich persönlich eher doof.

**Comment by Pakue95** [comment]
Die analoge Variante funktioniert auch wenn das WLAN mal nicht tut oder jemand das Homeassistant kaputt konfiguriert hat. Überwachen kannst du das ja immer noch über die Smartsteckdose.

**Comment by NitramLegov** [comment]
Die digitale Variante kann auch funktionieren, wenn WLAN und Homeassistant tot sind.

Mit Esphome ist das kein größeres Problem.

**Comment by NitramLegov** [comment]
Die Könighaus P-800 steht im Space und wartet darauf, commissioned zu werden. Dazu gibt es eine Wiki Seite:
https://wiki.comakingspace.de/K%C3%B6nighaus_P-800

Außerdem kann sie natürlich erst benutzt werden, wenn es eine Lösung zur Zeitbegrenzung gibt

**Comment by NitramLegov** [comment]
Im Sinne der Geschwindigkeit habe ich einen der Tasmotas, die im Space liegen auf ESPHome umgeflasht und so eingestellt, dass er automatisch nach einer einstellbaren Zeit ausgeht.

Die Zeit kann über HomeAssistant oder das WebInterface konfiguriert werden und läuft dann unabhängig von WLAN, etc.

Werde den demnächst in den Space bringen, dann können wir den zumindest so lange nutzen, bis eine der vorgeschlagenen anderen Lösungen bereit steht.

**Comment by ststefa** [comment]
Wir haben das Schätzchen in einer Gemsinschaftsaktion jetzt schon mal an die Wand gespaxt.
Wie gewünscht hochkant und in idealer Position ;-)

![Image](https://github.com/user-attachments/assets/81dc0c6d-7a6d-4e8a-b8a2-f5a73374fe20)

**Comment by NitramLegov** [comment]
Nachdem die Heizung jetzt ja schon draußen war haben wir jetzt auch einen smarten Zwischenstecker installiert, der die Heizung nach 10 Minuten automatisch ausmacht.

Fehlt dann noch etwas bezüglich der Raumbox?

**Comment by LukasFreeze** [comment]
Ja, wir sollten die kalte Zugluft bzw für eine mögliche Verwendung als Dunkelkammer auch Licht aussperren.

Mein letzter Stand war, dass @freegroup meinen Vorschlag mit einem Molton-Vorhang nicht so gut fand und eine Tür bauen wollte. Das Gespräch ist aber schon länger her, keine Ahnung wie aktuell dieser Plan noch ist...

Am Mittwoch lag ein Lamellenvorhang in der Box, der aber glaube ich nicht die beste Lösung wäre 🤔

**Comment by Neo-Johannes** [comment]
Gibt es für das Projekt noch eine verantwortliche Person? 2Ist die Raumbox eigentlich irgendwo am Boden oder mit einer Wand verschraubt/ankert? Und ist es immer noch der Plan ein Heizpilz reinzustellen? Ohne Heizpilz, E-Heizung oder ähnlichen sowie einer Tür (oder schweren Vorhang) ergibt die Raumbox für mich keinen Sinn, da die Infrarot-Heizung nicht die Luft bzw. den Raum aufwärmt. Die könnte man im Winter an anderen Stellen deutlich besser gebrauchen. Im Idealfall sogar ein Gestell mit Rollen geben, so dass man sie dort hin mitnehmen und einschalten kann, wo man sie braucht^^

**Comment by LukasFreeze** [comment]
Eine IR-Heizung heizt zwar nicht die Luft, trotzdem ist es mit kalter Zugluft wesentlich schwieriger damit eine angenehme Temperatur zu erreichen. Weitere Heizungsoptionen sind prinzipiell weiter denkbar, aber erst nachdem Tür oder Vorhang gebaut werden 🙃

Die Box war aber sowieso nicht nur als Heiz-Box gedacht, sondern sollte weitere Nutzungen beinhalten die sich damit nicht beißen und/oder verstärkt im Sommer stattfinden.

Auch dafür braucht es aber konkrete Ideen und Verantwortliche, aber da @freegroup sich seit meinem letzten Kommentar hier nicht gemeldet hat braucht es da wohl jemand anderes :-)
